docs/source/addons_and_starters/starters.md

```diff
@@ -0,0 +1,87 @@
+# Kedro starters
+
```

Comment:
Are we settle on the name `tool` for `add ons`?

Reply:
I think so.